packages/shared/components/molecules/NftActivityDetails.svelte

```diff
@@ -7,7 +7,7 @@
 
     import { Icon as IconEnum } from '@lib/auxiliary/icon'
 
-    export let activity: Activity
+    export let activity: Partial<Activity> = {}
```

Comment:
Nice change with the partial, but i would remove the default, as this is always required